langchain/scripts/create-entrypoints.js

```diff
@@ -138,7 +138,7 @@ const entrypoints = {
   "retrievers/supabase": "retrievers/supabase",
   "retrievers/zep": "retrievers/zep",
   "retrievers/metal": "retrievers/metal",
-  "retrievers/databerry": "retrievers/databerry",
```

Comment:
You'll need to keep the entrypoint in for `databerry` if you want back-compat (up to you!)